<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(152, 228, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bt_volver">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>3270 Nerd Font Mono</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(152, 228, 255);</string>
      </property>
      <property name="text">
       <string>Horario</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Carrera</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_carrera">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_carrera">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagenes/search.png</normaloff>imagenes/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Seccion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_sesion">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_sesion">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>imagenes/search.png</normaloff>imagenes/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_periodo">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Periodo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>9</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>113</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>59</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>07:30 A 08:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>08:10 A 08:50</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>08:50 A 09:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>09:30 A 10:10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10:10 A 10:50</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10:50 A 11:30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11:30 A 12:10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12:10 A 12:50</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12:50 A 1:30</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Lunes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Martes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Miercoles</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Jueves</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Viernes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sabado</string>
       </property>
      </column>
      <item row="0" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bt_guardar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:15px;
background-color:rgb(0, 169, 255);
color:black;</string>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_preview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:15px;
background-color:rgb(0, 169, 255);
color:black;</string>
        </property>
        <property name="text">
         <string>VistarPrevia</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
